csharp/examples/FluentBuilderExample/Program.cs

```diff
@@ -51,7 +51,7 @@ public static async Task Main(string[] args)
             using (var writer = new ArrowFileWriter(stream, recordBatch.Schema))
             {
                 await writer.WriteRecordBatchAsync(recordBatch);
-                await writer.WriteFooterAsync();
+                await writer.WriteEndAsync();
```

Comment:
This looks like a bad merge. Were you expecting to make a change here? Can this be reverted?

Reply:
I was planning to revert this, will mark this PR as a draft to flag this, sorry!